scripts/no-import.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
```

Comment:
What's the purpose of this script? Perhaps add docs to the top of the file?

Reply:
I used it to yank out import statements, but it was insufficient. Put there for posterity and visibility. Removed, now that the yanking has been complete.